src/v/config/configuration.h

```diff
@@ -55,6 +55,10 @@ struct configuration final : public config_store {
     property<std::size_t> coproc_max_batch_size;
     property<std::chrono::milliseconds> coproc_offset_flush_interval_ms;
 
+    // Controller
+    property<std::optional<std::size_t>> topic_memory_per_partition;
+    property<std::optional<int32_t>> topic_fds_per_partition;
```

Comment:
Aside: Do we have a decent way of marking config parameters as _unstable_? We're going to be adding a lot of little knobs that are really stop-gap solutions that should eventually go away. It would be nice to mark these as unstable to make the intention clearer.

Reply:
The distinction is between tunables (low level implementation-specific stuff) and full-fledged behavioural settings for users (https://vectorizedio.atlassian.net/wiki/spaces/CORE/pages/78676076/Configuration+Properties#Metadata-attributes).

The user facing docs for the concept of a tunable aren't written yet (next release) but when they are it'll explain the intrinsically ephemeral nature of properties that control an implementation detail -- tunables are basically all subject to deprecation at some point, when it's something that controls a buffer size or whatever, and one day that buffer might not exist any more.